docs/concepts-messaging.md

```diff
@@ -531,8 +534,7 @@ In the diagram below, **Consumer A**, **Consumer B** and **Consumer C** are all
 
 :::note
 
-**Limitations of Shared type**
- When using Shared type, be aware that:
+When using Shared type, be aware that:
```

Comment:
Suggest keeping consistent for the line 537 and 748

Reply:
I've paraphrased this sentence for simplicity. PTAL.